src/parser/shared/parserGenerator.ts

```diff
@@ -72,10 +75,9 @@ export function createParser(
             if (name) {
                 const [leaf, fieldName] = getObjectByPath(target, name);
 
-                if (isMultiple) {
-                    // prototype으로 디폴트값 넣어준 경우 nullish coalescing으로 해결 안됨 
-                    // @ts-ignore
-                    if (!Object.hasOwn(leaf, fieldName)) {
+                if (isMultiple && !isReducible) {
+                    // default value is injected via prototype, therefore have to check their own properties
+                    if (!Object.prototype.hasOwnProperty.call(leaf, fieldName)) {
```

Comment:
As I check `tsconfig.json`, the `target` was set to `ES2020` but `Object.hasOwn` is introduced in `ES2022`.
But the edge case of changing `Object.prototype.hasOwnProperty` is not that common, I think just using legacy one won't hurt general cases.